### PR TITLE
add listring to chest

### PR DESCRIPTION
**Pull request by DS-Minetest**

### CODE REVIEW

File: depends.txt [hunk 1]
```diff
@@ -1 +1 @@
-default
+default
```
Comment (numberZero) [inline depends.txt]: CR/LF?

**Comment by numberZero** [comment]
LGTM

File: depends.txt (hunk 1 above)
Comment (DS-Minetest) [inline depends.txt]: Idk what happened here nor what you mean with
>CR/LF?

.

Comment (numberZero) [inline depends.txt]: Linux, Win and Mac have different conventions of line ending. Namely, Win uses CR LF ("\r\n" in C), while Mac uses CR only, and Linux uses LF only. But here, that’s not the problem: I checked in my viewer; your version of `depends.txt` just lacks line ending character at all.

Comment (DS-Minetest) [inline depends.txt]: Interesting!
Shall I add a commit to make it like before?

Comment (numberZero) [inline depends.txt]: I think so.

Comment (DS-Minetest) [inline depends.txt]: Ok, if you think, it's a problem.

**Comment by DS-Minetest** [comment]
This PR is already waiting since months, is trivial, fixes something annoying and has a legitimisation.

**Comment by DS-Minetest** [comment]
To be reviewed/approved/merged.

**Comment by numberZero** [comment]
Who’s online? @kaeza?

**Comment by Hawk777** [comment]
Tested. Read the code. LGTM.

**Comment by Hawk777** [comment]
Was wondering, would it be appropriate to use a once-per-block LBM to rewrite all the existing chests’ formspecs? Or better to just require players to break and replace all their chests?

**Comment by DS-Minetest** [comment]
Listrings are just for comfort, an lbm would be too much.

**Pull request merged**